### PR TITLE
Update 22.4 version references to 23.0

### PR DESCRIPTION
**Pull request by Baezon**
Since these features were added before it was known the next version would be 23.0.

### CODE REVIEW

**Review by JohnAFernandez — APPROVED** [review]

**Comment by JohnAFernandez** [comment]
Thanks for finding these!

**Comment by JohnAFernandez** [comment]
Thanks for finding these!

**Review by Goober5000 — APPROVED** [review]
Okey dokey.

File: fred2/missionsave.cpp [hunk 1]
```diff
@@ -2778,7 +2778,7 @@ int CFred_mission_save::save_mission_info()
 		const auto& moveable_triggers = The_mission.skybox_model_animations.getRegisteredMoveables();
 
 		if (!anim_triggers.empty() || !moveable_triggers.empty()) {
-			fso_comment_push(";;FSO 22.4.0;;");
+			fso_comment_push(";;FSO 23.0.0;;");
```
Comment (BMagnu) [inline fred2/missionsave.cpp:2781]: At the _very_ least, this one has a mirror in qtfred.

**Review by BMagnu — CHANGES_REQUESTED** [review]
QT fred needs to be searched too

**Comment by Baezon** [comment]
Did a whole text search of the repo this time.

**Review by BMagnu — APPROVED** [review]
These _could_ be all.
I still fully expect we'll stumble across something unintentially missed later.
Also, did you change the wiki template?

**Comment by Baezon** [comment]
I did.

**Pull request merged**